src/faq/faqAccordion.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from 'react';
+import { Accordion } from 'hds-react';
+import { connect } from 'react-redux';
+import { RootState } from '../root/rootReducer';
+import { Faq } from './types';
+import { fetchFaqs } from './actions';
+import BlockLoader from '../loader/blockLoader';
+import { useTranslation } from 'react-i18next';
+
+interface State {
+  isFetchingFaqs: boolean;
+  faqs: Faq[];
+}
+
+interface Props {
+  isFetchingFaqs: boolean;
+  faqs: Faq[];
+  fetchFaqs: () => void;
+}
+
+const FaqAccordion = ({
+  isFetchingFaqs,
+  faqs,
+  fetchFaqs,
+}: Props): JSX.Element => {
+  useEffect(() => {
+    fetchFaqs();
+  }, []);
+
+  const { t } = useTranslation();
+
+  return (
+    <>
+      {isFetchingFaqs ? (
+        <BlockLoader />
+      ) : (
+        <Accordion
+          className="FaqAccordion"
+          heading={t('faq.accordion.heading', 'Frequently asked questions')}
+          initiallyOpen
+        >
+          {faqs.map((faq) => (
+            <div className="FaqItem" key={faq.question}>
```

Comment:
Ehh, this is a part of FaqAccordion and not a separate component so the classes should be subclasses of that all the way through.